### PR TITLE
add speech synthesis sample to generate srt subtitle file

### PR DESCRIPTION
**Pull request by yulin-li**
## Purpose

add speech synthesis sample to generate srt subtitle file

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

**Review by szhaomsft — DISMISSED** [review]

File: samples/java/jre/console/src/com/microsoft/cognitiveservices/speech/samples/console/SpeechSynthesisSamples.java [hunk 1]
```diff
@@ -695,6 +701,87 @@ else if (result.getReason() == ResultReason.Canceled) {
         synthesizer.close();
     }
 
+    // Synthesize and save audio and subtitle (in SubRip file format, SRT).
+    public static void synthesisWordBoundaryEventToSrtAsync() throws InterruptedException, ExecutionException, IOException {
```
Comment (dargilco) [inline samples/java/jre/console/src/com/microsoft/cognitiveservices/speech/samples/console/SpeechSynthesisSamples.java:705]: I don't see a change to main.java to add this new sample to the main menu

Comment (yulin-li) [reply samples/java/jre/console/src/com/microsoft/cognitiveservices/speech/samples/console/SpeechSynthesisSamples.java:705]: Yes... the main page use key A-Z to run samples and ran out of options... Maybe we need to refactor the menu to a two-level one, like what we did in python.

**Review by dargilco — DISMISSED** [review]

**Comment by BrianMouncer** [comment]
@yulin-li are you still working on this PR?

**Comment by yulin-li** [comment]
> @yulin-li are you still working on this PR?

Yes, still need this. I merged conflicts.

**Comment by yulin-li** [comment]
@BrianMouncer @dargilco Could you pls take a look?

**Review by dargilco — APPROVED** [review]

**Pull request merged**